### PR TITLE
Create 0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1.json

### PR DESCRIPTION
**Pull request by chillininvt**
Thank you

### CODE REVIEW

File: tokens/eth/0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "symbol": "DYL",
+  "address": "0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1",
+  "decimals": 18,
```
Comment (mattsk123) [inline tokens/eth/0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1.json:4]: "Amount"

Comment (chillininvt) [reply tokens/eth/0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1.json:4]: void token I believe this client was hacked through the thirdweb exploit.

Comment (mattsk123) [reply tokens/eth/0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1.json:4]: Wtf

**Review by mattsk123 — APPROVED** [review]

File: tokens/eth/0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1.json (hunk 1 above)
Comment (mattsk123) [reply tokens/eth/0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1.json:4]: > 19

**Pull request closed without merging**